build/gulp/plugins/gulp-component-menu.ts

```diff
@@ -27,12 +28,15 @@ export default () => {
     }
 
     try {
-      const info = getComponentInfo(file.path)
+      const infoFilename = file.basename.replace(/\.tsx$/, '.info.json')
 
-      if (info.isParent) {
+      const jsonInfo = fs.readFileSync(config.paths.docsSrc('componentInfo', infoFilename))
```

Comment:
maybe we could use a following semantics here, to prevent issue of tasks dependency:
- if file exists, use info from this file
- if not, use `getComponentInfo()` as fallback